src/bun.js/webcore/streams.zig

```diff
@@ -4104,7 +4101,8 @@ pub const FIFO = struct {
         this: *FIFO,
         buf: []u8,
     ) ReadResult {
-        switch (Syscall.read(this.fd, buf)) {
+        std.debug.print("DO READ\n", .{});
+        switch (Syscall.sys_uv.read(this.fd, buf)) {
```

Comment:
did you mean to keep this here?

Reply:
no i was trying to figure out where reading was happening because the todolog was disabled by default